src/views/Imprint.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Imprint() {
+  return (
+    <div>
+      <div className="mt-4 p-4">
+        <div className="impressum">
+          <strong>Impressum</strong>
+          <p>Angaben gemäß § 5 TMG</p>
+          <br />
+          <p>
+            Tom Graupner & Keno Dreßel
```

Comment:
Use explicit am percent so that browsers don't misinterpret it as the beginning of an entity.
```suggestion
            Tom Graupner &amp; Keno Dreßel
```